_includes/scenarios/integrate/access-the-connector.md

```diff
@@ -19,6 +64,10 @@ To fetch the Open API documentation of the Connector's REST API, visit the follo
 
 You can view these files with the [Swagger Editor](https://editor.swagger.io/) or automatically import them within your favorite API Clients (e.g. Postman or Insomnia).
 
-## API Authentication
+# SDK
+
+Preferably, an SDK should be used for integration of the connector into an application. The following SDKs are available for this purpose:
```

Comment:
Connector

Reply:
To achieve a better developer experience and type safety, preferably a Software Development Kit (SDK) should be used...

Reply:
We work on keeping this list as updated as possible. Please let us know, if some SDKs are outdated or new SDKs are available for the community.